sqlalchemy_redshift/__init__.py

```diff
@@ -1,14 +1,6 @@
-from pkg_resources import DistributionNotFound, get_distribution, parse_version
+from importlib.metadata import version
 
-for package in ['psycopg2', 'psycopg2-binary', 'psycopg2cffi']:
```

Comment:
It seems like we should put this constraint in the `setup.py` file in `install_requires`.

Reply:
I checked sqlalchemy 2 and psycopg2 >= 2.7 has been an optional constraint since at least sqla 2.0.